### PR TITLE
[API Shield] Sequence Mitigation + other fixes

### PR DESCRIPTION
**Pull request by patriciasantaana**

### CODE REVIEW

File: content/api-shield/security/jwt-validation/configure.md [hunk 1]
```diff
@@ -70,12 +70,13 @@ Refer to the example below to view the configuration using information required
...
-Example using cURL:
-
 ```json
-curl -X POST 'https://api.cloudflare.com/client/v4/zones/{zoneID}/api_gateway/token_validation' \
--H 'Content-Type: application/json' \
--d '{
+---
+header: Example using cURL
+---
+curl --request POST 'https://api.cloudflare.com/client/v4/zones/{zoneID}/api_gateway/token_validation' \
```
Comment (ranbel) [inline content/api-shield/security/jwt-validation/configure.md]:
```suggestion
curl POST 'https://api.cloudflare.com/client/v4/zones/{zoneID}/api_gateway/token_validation' \
```

Comment (ranbel) [reply content/api-shield/security/jwt-validation/configure.md]: according to https://developers.cloudflare.com/style-guide/api-content-strategy/guidelines-for-curl-commands/#request-with-json-body-content-post-put-patch

File: content/api-shield/security/schema-validation/configure.md [hunk 2]
```diff
@@ -51,8 +51,8 @@ Upload a schema via the v4 API using `POST`. This example requires a `example_sc
 ---
 header: cURL command
 ---
-curl -X POST "https://api.cloudflare.com/client/v4/zones/{zone_id}/api_gateway/user_schemas" \
-  -header "Authorization: Bearer <API_TOKEN>" \
+curl --request POST "https://api.cloudflare.com/client/v4/zones/{zone_id}/api_gateway/user_schemas" \
```
Comment (ranbel) [inline content/api-shield/security/schema-validation/configure.md]:
```suggestion
curl POST "https://api.cloudflare.com/client/v4/zones/{zone_id}/api_gateway/user_schemas" \
```

File: content/api-shield/security/schema-validation/configure.md [hunk 3]
```diff
@@ -51,8 +51,8 @@ Upload a schema via the v4 API using `POST`. This example requires a `example_sc
...
+  --header "Authorization: Bearer <API_TOKEN>" \
```
Comment (ranbel) [inline content/api-shield/security/schema-validation/configure.md]:
```suggestion
--header "Authorization: Bearer <API_TOKEN>" \
```

File: content/api-shield/security/schema-validation/configure.md [hunk 4]
```diff
@@ -51,8 +51,8 @@ Upload a schema via the v4 API using `POST`. This example requires a `example_sc
...
   --form file=@example_schema.yaml --form kind=openapi_v3 --form name=example_schema --form validation_enabled=false
```
Comment (ranbel) [inline content/api-shield/security/schema-validation/configure.md]:
```suggestion
--form file=@example_schema.yaml --form kind=openapi_v3 --form name=example_schema --form validation_enabled=false
```

Comment (ranbel) [reply content/api-shield/security/schema-validation/configure.md]: https://developers.cloudflare.com/style-guide/api-content-strategy/guidelines-for-curl-commands/#indentation

File: content/api-shield/security/schema-validation/configure.md [hunk 5]
```diff
@@ -88,10 +88,10 @@ Use a `PATCH` request to activate a schema after inspection.
 ---
 header: cURL command
 ---
-curl -X PATCH "https://api.cloudflare.com/client/v4/zones/{zone_id}/api_gateway/user_schemas/{schema_id}" \
-  -header "Authorization: Bearer <API_TOKEN>" \
-  -header 'Content-Type: application/json' \
-  -data '{
+curl --request PATCH "https://api.cloudflare.com/client/v4/zones/{zone_id}/api_gateway/user_schemas/{schema_id}" \
+  --header "Authorization: Bearer <API_TOKEN>" \
```
Comment (ranbel) [inline content/api-shield/security/schema-validation/configure.md]: May want to fix the indentation for the other curl commands on this page.

File: content/api-shield/security/schema-validation/configure.md [hunk 6]
```diff
@@ -189,10 +189,10 @@ You can add new operations in a schema to Endpoint Management using `POST`.
 ---
 header: cURL command
 ---
-curl -X POST "https://api.cloudflare.com/client/v4/zones/{zone_id}/api_gateway/operations" \
-  -header "Authorization: Bearer <API_TOKEN>" \
-  -header 'Content-Type: application/json' \
-  -data '[
+curl --request POST "https://api.cloudflare.com/client/v4/zones/{zone_id}/api_gateway/operations" \
```
Comment (ranbel) [inline content/api-shield/security/schema-validation/configure.md]:
```suggestion
curl POST "https://api.cloudflare.com/client/v4/zones/{zone_id}/api_gateway/operations" \
```

File: content/api-shield/security/schema-validation/configure.md [hunk 7]
```diff
@@ -229,12 +229,12 @@ You can add all operations in a schema that do not already exist in Endpoint Man
 ---
 header: cURL command
 ---
-curl -s -X POST "https://api.cloudflare.com/client/v4/zones/{zone_id}/api_gateway/operations" \
-  -header "Authorization: Bearer <API_TOKEN>" \
-  -header 'Content-Type: application/json' \
-  -data "(curl -s -X GET "https://api.cloudflare.com/client/v4/zones/{zone_id}/api_gateway/user_schemas/{schema_id}/operations?feature=schema_info" \
-  -header "Authorization: Bearer <API_TOKEN>" \
-  -header 'Content-Type: application/json' | jq ".result.new_operations")"
+curl -s --request POST "https://api.cloudflare.com/client/v4/zones/{zone_id}/api_gateway/operations" \
```
Comment (ranbel) [inline content/api-shield/security/schema-validation/configure.md]:
```suggestion
curl -s POST "https://api.cloudflare.com/client/v4/zones/{zone_id}/api_gateway/operations" \
```

File: content/api-shield/security/sequence-mitigation/_index.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+To illustrate, in the original sequence example listed in the [Configuration](/api-shield/security/sequence-mitigation/configure/) section, Sequence Mitigation would store the following sequence:
+
+1. `GET /api/v1/users/{user_id}/accounts`
+2. `GET /api/v1/accounts/{account_id}/balance`
+3. `POST /api/v1/transferFunds`
+
+Sequence Mitigation de-duplicated the two requests to `GET /api/v1/accounts/{account_id}/balance` and stored them as a single request.
+
+### Time Limitations
```
Comment (ranbel) [inline content/api-shield/security/sequence-mitigation/_index.md]:
```suggestion
### Time limitations
```

File: content/api-shield/security/sequence-mitigation/_index.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+Sequence Mitigation rules have a lookback period of ten minutes. If you create a rule that one path must be requested before another path and more than ten minutes elapses between a user requesting each path, the rule will not match.
```
Comment (ranbel) [inline content/api-shield/security/sequence-mitigation/_index.md]:
```suggestion
Sequence Mitigation rules have a lookback period of 10 minutes. If you create a rule that one path must be requested before another path and more than 10 minutes elapses between a user requesting each path, the rule will not match.
```

File: content/api-shield/security/sequence-mitigation/configure.md [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+    "last_updated": "2023-07-24T12:06:51.796286Z",
+    "created_at": "2023-07-24T12:06:51.796286Z"
+}
+```
+
+### Fields
+
+| Field name | Description | Possible Values | Example |
+| --- | --- | --- | --- |
+| `id` | An opaque identifier that identifies a rule.  | a UUID |  `"d4909253-390f-4956-89fd-92a5b0cd86d8"` |
```
Comment (ranbel) [inline content/api-shield/security/sequence-mitigation/configure.md]:
```suggestion
| `id` | An opaque identifier that identifies a rule.  | A UUID |  `"d4909253-390f-4956-89fd-92a5b0cd86d8"` |
```

File: content/api-shield/security/sequence-mitigation/configure.md [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+| `title` | A string that helps to identify the rule. | A value between 1 and 50 characters | "Allow checkout sequence" |  
+| `kind` | Defines the semantics of this rule. Blocks rules have a negative security model and allow to explicitly deny a sequence. Allow rules have a positive security model and deny everything but the configured sequence. | `block`, `allow` | `"block"` |
```
Comment (ranbel) [inline content/api-shield/security/sequence-mitigation/configure.md]:
```suggestion
| `kind` | Defines the semantics of this rule. Block rules have a negative security model and allow to explicitly deny a sequence. Allow rules have a positive security model and deny everything but the configured sequence. | `block`, `allow` | `"block"` |
```

File: content/api-shield/security/sequence-mitigation/configure.md [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+| `action` | What firewall action should we do when the rule matches. | `block`,`log` | `"log"` |
+| `sequence` | Denotes the operations (from Endpoint Management) that make up the sequence for this rule. We currently only support sequences of length 2. | An array with two valid Operation IDs from Endpoint Management |`["0d9bf70c-92e1-4bb3-9411-34a3bcc59003", "b704ab4d-5be0-46e0-9875-b2b3d1ab42f9"]` |
```
Comment (ranbel) [inline content/api-shield/security/sequence-mitigation/configure.md]:
```suggestion
| `sequence` | Denotes the operations (from Endpoint Management) that make up the sequence for this rule. We currently only support sequences of length two. | An array with two valid Operation IDs from Endpoint Management |`["0d9bf70c-92e1-4bb3-9411-34a3bcc59003", "b704ab4d-5be0-46e0-9875-b2b3d1ab42f9"]` |
```

File: content/api-shield/security/sequence-mitigation/configure.md [hunk 13]
```diff
@@ -0,0 +1,147 @@
...
+    "created_at": "2023-07-24T12:06:51.796286Z"
+}
+```
+
+### Fields
+
+| Field name | Description | Possible Values | Example |
+| --- | --- | --- | --- |
+| `id` | An opaque identifier that identifies a rule.  | a UUID |  `"d4909253-390f-4956-89fd-92a5b0cd86d8"` |
+| `title` | A string that helps to identify the rule. | A value between 1 and 50 characters | "Allow checkout sequence" |
```
Comment (ranbel) [inline content/api-shield/security/sequence-mitigation/configure.md]: should all examples be in backticks?

File: content/api-shield/security/sequence-mitigation/configure.md [hunk 14]
```diff
@@ -0,0 +1,147 @@
...
+
+### List sequence rules
+
+Use the `GET` command to list rules. 
+
+```bash
+---
+header: cURL command
+---
+curl --request GET 'https://api.cloudflare.com/client/v4/zones/{zone_id}/api_gateway/seqrules’
```
Comment (ranbel) [inline content/api-shield/security/sequence-mitigation/configure.md]:
```suggestion
curl GET 'https://api.cloudflare.com/client/v4/zones/{zone_id}/api_gateway/seqrules’
```

File: content/api-shield/security/sequence-mitigation/configure.md [hunk 15]
```diff
@@ -0,0 +1,147 @@
...
+
+This adds a single rule to all existing rules. Priority can be used to place the rule between, before, or after another rule.
+
+The response will reflect the rule that has been written with its ID. In case something is not right with the rule, an appropriate error message with a `json` path pointing towards the issue will be provided.
+
+```json
+---
+header: Example using cURL
+---
+curl --request POST 'https://api.cloudflare.com/client/v4/zones/{zoneID}/api_gateway/token_validation' \
```
Comment (ranbel) [inline content/api-shield/security/sequence-mitigation/configure.md]:
```suggestion
curl POST 'https://api.cloudflare.com/client/v4/zones/{zoneID}/api_gateway/token_validation' \
```

**Review by ranbel — APPROVED** [review]

File: content/api-shield/security/jwt-validation/configure.md (hunk 1 above)
Comment (pedrosousa) [reply content/api-shield/security/jwt-validation/configure.md]: My bad! Actually, I meant "do not include the `--request POST` command-line argument".

I've updated the curl guidelines and I will update these occurrences on a separate PR.

Comment (pedrosousa) [reply content/api-shield/security/jwt-validation/configure.md]: Clarified the guidelines in this PR: https://github.com/cloudflare/cloudflare-docs/pull/10114

Proposed some curl changes related to the current PR in the following GitHub PR: https://github.com/cloudflare/cloudflare-docs/pull/10119. Has a few notes about the changes I'm proposing that may be useful.

**Pull request merged**